types/jest/index.d.ts

```diff
@@ -863,7 +866,7 @@ declare namespace jest {
         testRunner: string;
         testURL: string;
         timers: 'real' | 'fake';
-        transform: Array<[string, Path]>;
```

Comment:
This is how jest defined in their types: https://github.com/facebook/jest/blob/eb096f8b8d4a0db77fbce112fd758af7a6496764/types/Config.js#L270

I guess `InitialOptions` should be what you're looking for?
https://github.com/facebook/jest/blob/eb096f8b8d4a0db77fbce112fd758af7a6496764/types/Config.js#L157

Reply:
wow, that's super confusing naming in jest codebase indeed. But you are correct. 
Anyway we don't have `InitialOptions` interface here. So should I add it  ( mirror jest types in our TS types ) ? 

thanks